qucs/qucs/mouseactions.cpp

```diff
@@ -319,6 +356,8 @@ void MouseActions::MMoveElement(Schematic *Doc, QMouseEvent *Event)
  */
 void MouseActions::MMoveWire2(Schematic *Doc, QMouseEvent *Event)
 {
+//ALYS - i saved old code inside #if 0 block
```

Comment:
> is it good?

no. you can add more commits to this PR. just push them to the branch.

you may even edit your local commits and force push. this is what you want to do just before a merge. please post a note such as "rebased".

> do not know who is author

use `git blame`

> May be delete them too?

only if they are close by/related. otherwise use a dedicated PR for cleanup